targets/stm32l432/Makefile

```diff
@@ -9,6 +9,25 @@ merge_hex=../../tools/solotool.py mergehex
 
 .PHONY: all all-hacker all-locked debugboot-app debugboot-boot boot-sig-checking boot-no-sig build-release-locked build-release build-release build-hacker build-debugboot clean clean2 flash flash_dfu flashboot detach cbor test
```

Comment:
@conorpp Please add the new targets to the .PHONY line as the are not updateable files.
for example:
```
.PHONY: all all-hacker all-locked debugboot-app debugboot-boot boot-sig-checking boot-no-sig build-release-locked build-release build-release build-hacker build-debugboot clean clean2 flash flash_dfu flashboot detach cbor test firmware-hacker firmware-secure bootloader-nonverifying bootloader-verifying full-clean
```

Reply:
Will do, will add in future commit